File: src/main/java/com/google/android/finsky/api/DfeResponseVerifier.java
package com.google.android.finsky.api;

public interface DfeResponseVerifier
{
  String getSignatureRequest()
    throws DfeResponseVerifier.DfeResponseVerifierException;

  void verify(byte[] paramArrayOfByte, String paramString)
    throws DfeResponseVerifier.DfeResponseVerifierException;

  class DfeResponseVerifierException extends Exception
  {
    public DfeResponseVerifierException(String paramString)
    {
      super();
    }
  }
}

File: playstore/src/main/java/finsky/api/DfeResponseVerifier.kt
package finsky.api

interface DfeResponseVerifier {
    val signatureRequest: String

    @Throws(DfeResponseVerifier.DfeResponseVerifierException::class)
    fun verify(paramArrayOfByte: ByteArray, paramString: String)

    class DfeResponseVerifierException(val errorMessage: String) : Exception()
}